types/FluentJSONSchema.test-d.ts

```diff
@@ -110,7 +110,12 @@ const rawNullableSchema = S.object()
 
 console.log('raw schema with nullable props\n', JSON.stringify(rawNullableSchema))
 
-const dependentRequired = S.object()
+type Foo = {
+  foo: string
+  bar: string
+}
+
+const dependentRequired = S.object<Foo>()
```

Comment:
Can you duplicate another set of test case instead of modifying the old one.
It can ensure we support the case that didn't pass generic param.

Reply:
Yes, of course.